app/views/orders/confirmation.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1> Congratulations! You bought some stuff!</h1>
+<div class="checkout">
+<h2>Details of Order # <%= @order.id %><br></h2>
+Sold To: <%= @order.name_on_credit_card %><br>
+Shipping To:
+<%= @order.street_address %><br>
+<%= @order.city %><br>
+<%= @order.state %>, <%= @order.billing_zip %><br>
+Credit Card : **** **** **** **** <%= @order.credit_card_number %><br><br>
+<% @cart_price = 0 %>
+<% @order.orderitems.each do |item| %>
+Item: <%= link_to Product.find(item.product_id).name , product_path(Product.find(item.product_id).id) %><br>
+Quantity: <%= item.quantity %><br>
+Subtotal: $<%= number_with_precision item.total_price , precision: 2%><br><br>
+<% @cart_price += item.total_price %>
+<% end %>
+
+<h3> Total Price of Order: $<%= number_with_precision @cart_price , precision: 2   %></h3>
+<h3> Order Status: <%= @order.status %></h3>
+<h3> Time Order was placed: <%= @order.updated_at %></h3>
+
+<%= link_to "Thanks for Shopping at Petsy!", root_path %>
+</div>
```

Comment:
Not sure if this is github messing things up, but make sure you're indenting your html correctly.